### DISCUSSION TITLE
Wican bricked?

### DISCUSSION THREAD

**Discussion by KlausHans** [meatpiHQ/wican-fw#182] [General]
Hello @meatpiHQ,

i tried to install the v3.25Beta yesterday. Since then the Wican is not available any more. There is also no AP and only the Wifi led flashes every second.
Is it bricked? Can it be undone?

**Comment by meatpiHQ** [comment]
@KlausHans what was your firmware version before the update?

**Reply by KlausHans** [reply]
3.10 Beta

**Reply by KlausHans** [reply]
With 3.10 i had the problem, that the Wican didn't go into sleep mode, that's why i wanted to try the latest one.

**Reply by meatpiHQ** [reply]
Oh someone else had he same issue, please recover using the USB port. Make sure to erase before you flash otherwise it won't work.


https://github.com/meatpiHQ/wican-fw?tab=readme-ov-file#2-usb-flash

**Reply by KlausHans** [reply]
Thanks, i will try it.

**Reply by KlausHans** [reply]
Works again, update to v3.25Beta also succeeded. 👍

**Reply by TheSmartGerman** [reply]
the same happend to me, updating from 3.10 to 3.25beta seems to be a little tricky.

**Reply by typxxi** [reply]
thanks for sharing 
is 3.25 beta a new beta ?
I had tried one which was beyond 3.11 beta but I can not remember.

UPDATE: I checked it and it must have been 3.25 as it has been released a week ago or so . I was able to get there (3.25beta) and back to 2.98 - somehow , can't remember except I had to upload the firmware from my local drive.

**Comment by TheSmartGerman** [comment]
Well, there is also a 3.26beta. some debug features and Homeassistent discovery fixes, if i'm not wrong 

Am 27. August 2024 17:13:16 MESZ schrieb Wolf Gang ***@***.***>:
